### DISCUSSION TITLE
How to slow down the track and transpose pitches

### DISCUSSION THREAD

**Discussion by ljfreelancer88** [CoderLine/alphaTab#1168] [Q&A]
Hi,

How to slow down the track and transpose pitches? I'm using the code from the tutorial/documentation as my base template.

**Comment by Danielku15** [comment]
What do you mean exactly with "slow down" in this context? Could it be a translation issue of your question? 
There is a setting which controls the transposition pitches: https://alphatab.net/docs/reference/settings/notation/transpositionpitches#description

And as you have found already [this PR](https://github.com/CoderLine/alphaTab/pull/957/files) in 1.3 (release date not known) some more on-the-fly changes will be possible.

**Reply by ljfreelancer88** [reply]
This is what I'm referring [https://alphatab.net/docs/reference/settings/notation/transpositionpitches/#description](https://alphatab.net/docs/reference/settings/notation/transpositionpitches/#description). I guess I have to wait for the release.
